test/unit/api/fittings/lisk_cors.js

```diff
@@ -87,9 +87,9 @@ describe('lisk_cors', function () {
 
 		cors_fititng(context, next);
 
-		next.should.have.not.been.called;
-		context.response.statusCode.should.be.equal(204);
-		should.not.exist( context.response.getHeader('my-custom-header') );
+		expect(next).to.have.not.been.called;
+		expect(context.response.statusCode).to.be.equal(204);
+		expect( context.response.getHeader('my-custom-header') ).to.be.undefined;
```

Comment:
Did you determine these individually? Also spacing is strange.

Reply:
Yes ran the test to check expected value